hw03_sethutsa.py

```diff
@@ -0,0 +1,197 @@
+#################################################################################
+# Author: Utsa Seth
+# Username: sethutsa
+#
+# Assignment: HW 03
+# Purpose: Make a turtle artwork based on guidelines
+# Google Doc Link: https://docs.google.com/document/d/1eaKOWOABEuIIrmJ1DmS89uaWYTad68SuiMbo4g9Pe5M/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: Learned how to add images and text from t03_functions_house.py
+# provided with t03 assignment. Took the idea of changing Dory's "just keep swimming"
+# to "just keep coding" from Nicholas Hamilton.
+#
+#################################################################################
+import turtle
+
+
+def draw_fish(t):
```

Comment:
Missing a docstring

Reply:
Actually, the docstring is in the wrong spot (it's at the end of every function, instead of the start).